source/sync/data-model/update-data-model/update-schema-production-apps.txt

```diff
@@ -1,20 +1,400 @@
+
+Version Compatibility
+~~~~~~~~~~~~~~~~~~~~~
+
+Schema versions are considered either compatible or incompatible with previous
+schemas, depending on the type of change made: 
+
+- **Compatible:** When you make a schema versioning change, the new schema
+  version is backward-compatible with all previous schema versions to the
+  current minimum version, and forward-compatible until you make a breaking change.
+
+- **Incompatible:** When you make a breaking change, the new schema version is
+  *not* backward-compatible with any previous schema versions. An incompatible
+  schema version invalidates all previous schemas and effectively becomes the
+  new minimum version. 
+
+Schema Versioning and Development Mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To use schema versioning, you must disable Development Mode. You *cannot* use
+both at the same time in the same app. To learn how to disable Development Mode
+in the Sync Settings, refer to :ref:`Disable Development Mode
+<enable-development-mode>`.
+
+When you disable Development Mode, schema versioning is automatically enabled
+for eligible apps.
+
+If you enable Development Mode in an app with existing schema versions, schema
+versioning is automatically disabled. App Services deletes all existing schema
+versions and resets the current version to 0. You cannot create incremented
+versions again until Development Mode is disabled. 
+
+.. warning:: Enabling Development Mode Can Break Versioned Clients
+
+   Because existing schema versions are deleted when you enable Development
+   Mode, any clients that use the deleted schemas will break.
+
+   .. TODO: Q. What do clients need to do in this case? Clear the schemaVersion from syncConfig?
+
+Schema Versioning and Terminating Sync 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can :ref:`terminate and re-enable Device Sync <terminating-realm-sync>`
+without affecting schema versioning. All existing schema versions are retained.
+You can also make changes while Sync is terminated, and the schema version
+increments as expected. 
+
+However, if you update your data model while Device Sync is terminated, any
+changes that you make are considered breaking changes, even if they are
+normally considered non-breaking or schema versioning changes. 
+
+Change Types
+------------
+
+When schema versioning is enabled, changes are categorized as one of three types:
+
+- **Non-Breaking (Additive):** Changes that do *not* create a new schema
+  version, unless they are applied with schema versioning or breaking changes. 
+
+  Instead of creating a new schema, additive changes are automatically applied to the current schema version. This means that only clients with a schema version equal to or greater than that version can reference the added element.
+  
+  Additive changes include:
+
+  - Adding a new object type or property
+  - Adding a default value on a property
```

Comment:
afaik we refer to these as _object types_ and _properties_ in the docs, rather than _tables_ and _fields_. I can split those out into separate bullets, though, if it's not clear those are two different actions: 
```suggestion
  - Adding a new object type 
  - Adding a new property
```